lib/straw/omd_util.ml

```diff
@@ -28,8 +28,8 @@ let split_links attr (tag, r) text =
         { Omd.title; label; destination = "#" })
     | Str.Text (s) -> Omd.Text (attr, s))
 
-let mention_regexp = Str.regexp {|@\([^@ .,;:!?\t\n]+\)|}
```

Comment:
I have

Reply:
It seems strange to me that these aren't handled. You mentioned the other day a source that said that whitespace characters aren't handled by Str, can you link it?

Reply:
I said something about the regex not being able to span multiple lines. I base this statement on the following: 
![2023-12-29-175047_600x56_scrot](https://github.com/bas080/straw/assets/1840380/11db2a46-8470-47d5-ae7d-78f93566a7c1)
f

> https://v2.ocaml.org/api/Str.html#VALsplit

Reply:
The source you sent me says:

> For instance, split (regexp "[ \t]+") s splits s into blank-separated words.

It supports \t and it also supports \n, however you'll notice that it does "\t" in string quotes, which means it will be interpreted as a literal tab and it won't be escaped. 

I'm using `{| ... |}` here, which is the multiline string operator. Try doing  `"@\\([^@ .,;:!?\t\n]+\\)"` instead perhaps?

Reply:
Also, the "Matches any character except newline" is talking about `.`, which doesn't mean that the regexes can't handle a newline.

Reply:
That seems to have worked. I made a small addition. Won't allow `#@` and `@#` as mention or hashtag.